signac/contrib/project.py

```diff
@@ -424,6 +430,7 @@ def __contains__(self, job):
         """
         return job.get_id() in self.find_job_ids()
```

Comment:
Let's replace the internal uses of `find_jobs_ids()` with `_find_job_ids()`.
```suggestion
         return job.get_id() in self._find_job_ids()
```

Reply:
Other places where it's used that I see: `open_job`, `min_len_unique_id`, `check`, `repair`, `_build_index`, and in `JobsCursor.__len__` and `JobsCursor.__iter__`. Should just require adding the underscore once you add the new method.